perf/renaissance-jmh/playlist.xml

```diff
@@ -0,0 +1,293 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!--
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+-->
+<playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../TKG/playlist.xsd">
+	<include>../perf.mk</include>
+	<test>
+		<testCaseName>renaissance-jmh-AkkaUct</testCaseName>
+		<command>
+			$(JAVA_COMMAND) $(ADD_OPENS_CMD) $(JVM_OPTIONS) -Xlog:gc*:file=AkkaUct_gc.log -XX:+UseG1GC -Xms12G -Xmx12G -XX:ThreadPriorityPolicy=1 -XX:+AlwaysPreTouch -XX:+UseLargePages -XX:+UseTransparentHugePages -jar $(Q)$(TEST_RESROOT)$(D)renaissance-jmh.jar$(Q) -f 1 -prof gc -bm avgt -rf json -rff $(Q)$(REPORTDIR)$(D)AkkaUct.json$(Q) AkkaUct; $(TEST_STATUS)
```

Comment:
There's a lot of duplicate code in these commands:

e.g `XX:+UseG1GC -Xms12G -Xmx12G -XX:ThreadPriorityPolicy=1 -XX:+AlwaysPreTouch -XX:+UseLargePages -XX:+UseTransparentHugePages` is consistent across all test targets, perhaps it could be generated using the same approach as the `ADD_OPENS_CMD`?

Reply:
Those flags are consistent across all test targets, but they're not necessarily going to be consistent across all benchmarks. Are you suggesting I add a Renaissance-JMH-specific list of JVM options to perf.mk (e.g., RENAISSANCE_JMH_JVM_OPTIONS) and use that in gen_playlist.py when I generated playlist.xml? Functionally I'm not sure how different that is than simply specifying the list of options once in the gen_playlist.py script.

Reply:
agreed, let's leave them hard coded in the generator for now